checkout/session/client.go

```diff
@@ -81,22 +110,7 @@ func (c Client) ListLineItems(id string, listParams *stripe.CheckoutSessionListL
 	})}
 }
 
-// Iter is an iterator for sessions.
-type Iter struct {
-	*stripe.Iter
-}
-
-// CheckoutSession returns the session which the iterator is currently pointing to.
-func (i *Iter) CheckoutSession() *stripe.CheckoutSession {
-	return i.Current().(*stripe.CheckoutSession)
-}
-
-// CheckoutSessionList returns the current list object which the iterator is
-// currently using. List objects will change as new API calls are made to
-// continue pagination.
-func (i *Iter) CheckoutSessionList() *stripe.CheckoutSessionList {
-	return i.List().(*stripe.CheckoutSessionList)
-}
+type LineItemIter lineitem.Iter
```

Comment:
Does this make it a breaking change on existing integrations?
Asking as there's an `Iter` in `lineitem.go` and some broken tests at the bottom

Reply:
You're right, this is breaking as is -- we need to do `type LineItemIter = lineitem.Iter` to make it a transparent type alias and that will fix the test. I thought I had written the correct override for codegen to do this, need to debug.